ovos_plugin_manager/templates/audio.py

```diff
@@ -58,52 +63,143 @@ def runtime_requirements(self):
                                    no_internet_fallback=True,
                                    no_network_fallback=True)
 
-    # methods below deprecated and handled by OCP directly
-    # playlists are no longer managed plugin side
-    # this is just a compat layer forwarding commands to OCP
+    @property
+    def playback_time(self):
+        return 0
+
+    @abstractmethod
+    def supported_uris(self):
+        """List of supported uri types.
+
+        Returns:
+            list: Supported uri's
+        """
+
+    @abstractmethod
     def clear_list(self):
         """Clear playlist."""
```

Comment:
this one should not be reverted, the default code is valid and does not need to be an abstract method

Reply:
reverted with try/except